src/Queue/Worker.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace CacheWerk\BrefLaravelBridge\Queue;
+
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Queue\WorkerOptions;
+use Illuminate\Queue\Worker as LaravelWorker;
+
+class Worker extends LaravelWorker
```

Comment:
Will the logging still happen via Laravel?

https://github.com/cachewerk/bref-laravel-bridge/pull/11

Reply:
Added again